gateway-service/src/main/java/org/zowe/apiml/gateway/security/service/schema/HttpBasicPassTicketScheme.java

```diff
@@ -125,12 +131,18 @@ public static class PassTicketCommand extends AuthenticationCommand {
         String cookieName;
         String patCookieName;
         Long expireAt;
+        String customHeader;
 
         @Override
         public void apply(InstanceInfo instanceInfo) {
             if (authorizationValue != null) {
                 final RequestContext context = RequestContext.getCurrentContext();
                 context.addZuulRequestHeader(HttpHeaders.AUTHORIZATION, authorizationValue);
+                if (StringUtils.isNotEmpty(customHeader)) {
+                    String passTicket = authorizationValue.replace("Basic ", "");
```

Comment:
The passTicket variable will actually hold encoded username+passticket ... is it what should be put in the special header?

Reply:
Addressed it